### PR TITLE
Copy for completed intake with no EP created

### PR DESCRIPTION
**Pull request by leikkisa**
connects #8724

### CODE REVIEW

**Comment by leikkisa** [comment]
screen shot 2019-01-16 at 6 30 13 pm

File: spec/feature/intake/confirmation_page_spec.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+  before { setup_intake_flags }
+  after { teardown_intake_flags }
+
+  let!(:current_user) { User.authenticate!(roles: ["Mail Intake"]) }
+  let(:before_ama_date) { 3.years.ago }
+
+  describe "when completing a claim review" do
+    describe "Confirmation copy" do
+      [:higher_level_review, :supplemental_claim].each do |claim_review_type|
```
Comment (pkarman) [inline spec/feature/intake/confirmation_page_spec.rb]: I like that we are trying to re-use logical flow as much as we can!

File: spec/feature/intake/confirmation_page_spec.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            add_untimely_exemption_response("Yes") if claim_review_type == :higher_level_review
+            click_intake_finish
+            expect(page).to have_content("Intake completed")
+            expect(page).to_not have_content("If you need to edit this, go to VBMS claim details")
+            expect(page).to_not have_content("Informal Conference Tracked Item")
+            expect(page).to have_content("Edit the notice letter to reflect the status of requested issues")
+          end
+
+          it "shows EP related content if there is an end product created" do
+            if claim_review_type == :supplemental_claim
```
Comment (pkarman) [inline spec/feature/intake/confirmation_page_spec.rb]: this seems ripe for a refactor into its own method `start_claim_review(claim_review_type, veteran)`

File: spec/feature/intake/intake_edit_confirmation_spec.rb [hunk 3]
```diff
@@ -109,6 +124,22 @@
...
         expect(page).to have_content("Informal Conference Tracked Item")
       end
+
+      it "does not show informal conference request if there are no end products" do
+        decision_review.update!(informal_conference: true)
+
+        visit edit_path
+        click_intake_add_issue
+        click_intake_no_matching_issues
+        add_intake_nonrating_issue(date: (decision_review.receipt_date - 2.years).strftime("%m/%d/%Y"))
```
Comment (pkarman) [inline spec/feature/intake/intake_edit_confirmation_spec.rb]: I think `strftime("%D")` also works

File: spec/feature/intake/confirmation_page_spec.rb (hunk 2 above)
Comment (leikkisa) [reply spec/feature/intake/confirmation_page_spec.rb]: 👍

File: spec/feature/intake/intake_edit_confirmation_spec.rb (hunk 3 above)
Comment (leikkisa) [reply spec/feature/intake/intake_edit_confirmation_spec.rb]: 👍

**Pull request merged**